gitlab/resource_gitlab_service_slack.go

```diff
@@ -160,49 +182,20 @@ func resourceGitlabServiceSlack() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 			},
-			"job_events": {
-				Description: "Enable notifications for job events.",
+			"wiki_page_events": {
+				Description: "Enable notifications for wiki page events.",
 				Type:        schema.TypeBool,
+				Optional:    true,
 				Computed:    true,
 			},
 		},
 	}
 }
 
-func resourceGitlabServiceSlackSetToState(d *schema.ResourceData, service *gitlab.SlackService) error {
-	d.SetId(fmt.Sprintf("%d", service.ID))
-	d.Set("webhook", service.Properties.WebHook)
-	d.Set("username", service.Properties.Username)
-	d.Set("notify_only_broken_pipelines", bool(service.Properties.NotifyOnlyBrokenPipelines))
-	d.Set("notify_only_default_branch", bool(service.Properties.NotifyOnlyDefaultBranch))
-	d.Set("branches_to_be_notified", service.Properties.BranchesToBeNotified)
-	d.Set("push_events", service.PushEvents)
-	d.Set("push_channel", service.Properties.PushChannel)
-	d.Set("issues_events", service.IssuesEvents)
-	d.Set("issue_channel", service.Properties.IssueChannel)
-	d.Set("confidential_issues_events", service.ConfidentialIssuesEvents)
-	d.Set("confidential_issue_channel", service.Properties.ConfidentialIssueChannel)
-	d.Set("merge_requests_events", service.MergeRequestsEvents)
-	d.Set("merge_request_channel", service.Properties.MergeRequestChannel)
-	d.Set("tag_push_events", service.TagPushEvents)
-	d.Set("tag_push_channel", service.Properties.TagPushChannel)
-	d.Set("note_events", service.NoteEvents)
-	d.Set("note_channel", service.Properties.NoteChannel)
-	d.Set("confidential_note_events", service.ConfidentialNoteEvents)
-	// See comment to "confidential_note_channel" in resourceGitlabServiceSlack()
-	//d.Set("confidential_note_channel", service.Properties.ConfidentialNoteChannel)
-	d.Set("pipeline_events", service.PipelineEvents)
-	d.Set("pipeline_channel", service.Properties.PipelineChannel)
-	d.Set("wiki_page_events", service.WikiPageEvents)
-	d.Set("wiki_page_channel", service.Properties.WikiPageChannel)
-	d.Set("job_events", service.JobEvents)
-
-	return nil
-}
-
 func resourceGitlabServiceSlackCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	client := meta.(*gitlab.Client)
 	project := d.Get("project").(string)
+	d.SetId(project)
```

Comment:
This is a breaking change, `d.SetId(fmt.Sprintf("%d", service.ID))` should have been preserved.